src/test/resources/org/prebid/server/ApplicationTest/openrtb2/somoaudience/test-somoaudience-bid-request-1.json

```diff
@@ -60,40 +55,6 @@
     }
   },
   "ext": {
```

Comment:
Please modify auction request so that it contains 4 imps: 2 banner imps with different placement hashes, 1 video and 1 native imps.